### ISSUE TITLE
Instructions: Getting vision working on DICE

### ISSUE THREAD

**Issue by easyCZ** [Tasignotas/SDP#8]
Ideally, we'd have a build script that we can run on a dice machine and have it do all the hard work of changing the paths.

@RyanrDavies, @rowanborder, @victor-dumitrescu  You have the most experience with it, could you write the instructions?

**Comment by rowanborder** [comment]
Which paths need changing exactly when setting up simpleCV on DICE?

**Comment by grantkelly** [comment]
I thought the problem we had with SimpleCV was that PyGame was a dependency we didn't have. It's OpenCV that Ryan et co are using at the moment.

You don't have to change any paths to install openCV, which is what we're using at the moment. If you follow the instructions for installation here http://docs.opencv.org/trunk/doc/py_tutorials/py_setup/py_setup_in_fedora/py_setup_in_fedora.html and proceed to build it in your home directory, it will work fine.

The only problem we encountered was that our disk quota had been exceeded. Clearing unnecessary files fixes the problem. When performing the build, using CMake with these flags:

cmake -D BUILD_DOCS=ON -D BUILD_TESTS=OFF -D BUILD_PERF_TESTS=OFF -D BUILD_EXAMPLES=OFF ..

will build a smaller version, with documentation but no examples or tests. It fits easily in the home directory (but remember to build it inside the clone folder you made).

**Comment by rowanborder** [comment]
Is it worth creating a build script for the simpleCV installation then?

**Comment by easyCZ** [comment]
I tried using Grant's build script but still can't get it to build. I end up with permissions issue each time. Perhaps one of you could have a look at it tomorrow.

**Comment by rowanborder** [comment]
What directory is the permissions issue for specifically?

**Comment by easyCZ** [comment]
Okay, I did the following:
1. use wget to download OpenCV
2. Extract to /disc/scrap/sdp/group7/
3. Call cmake with all those fancy params and PREFIX set to '~/.local'
4. make
5. make install

When 5. terminates, I get an error stating I do not have sufficient permission to install. 
I used scrap to avoid any quota issues and assumed that setting the prefix will actually compile it to ./local.

I tried Grant's script too but ended up getting permission error when doing 'make install'
